aldryn_search/utils.py

```diff
@@ -134,19 +136,14 @@ def strip_tags(value):
     strip tags. If value isn't valid, just return value since there is
     no tags to strip.
     """
-    # borrowed from aldryn-search
-
     if isinstance(value, six.string_types):
         value = value.strip()
 
-        if not value:
```

Comment:
Yes, it raises `lxml.etree.XMLSyntaxError`, wich use as base `lxml.etree.ParseError`. That's why I used ParseError here. I'm trying to get another parsing errors too.